premis_event_service/migrations/0002_add_event_ordinal.py

```diff
@@ -144,7 +144,7 @@ class Migration(migrations.Migration):
                 name='event_identifier',
                 field=models.CharField(
                     help_text=(
-                        b'Unique identifier for an event. '
```

Comment:
I am curious about this being changed from bytest to text here, but not in 0001 migration. Is there a reason for that?

Reply:
I suppose since the migrations have been brought up, something I have been wanting to mention is that I really don't think we _need_ to modify them at all. They're made to be created and left alone, regardless of what version of Python you're on. The future imports don't mess anything up in Python 3+, so I think honestly the best thing would be to just ignore migrations completely, unless there is a specific reason that one needs to be modified.

Reply:
That sounds ok to me it that is the case.

Reply:
Well, it gives a syntax error as the line below it doesn't start with `b` prefix. I chose to remove `b` from this line instead of adding it to the next line 'cos I thought it's not necessary to have `help_text` to be in bytes.

Reply:
Ok, so I guess this is the only place it was a multi-line string then.

Reply:
yes. It looks like the other migration files are all using bytes so I can add `b` prefix to the second line for consistency.

Reply:
it's ok, I think text makes more sense anyway. You can leave it.